app/util/password/index.ts

```
@@ -2,9 +2,8 @@ import SecureKeychain from '../../core/SecureKeychain';
 import Engine from '../../core/Engine';
 
 export const MIN_PASSWORD_LENGTH = 8;
-type PasswordStrength = 0 | 1 | 2 | 3 | 4;
 
-export const getPasswordStrengthWord = (strength: PasswordStrength) => {
+export const getPasswordStrengthWord = (strength: number) => {
```

This would provide no type warning if a number other than 0-4 was passed in - it would fail silently. Perhaps it would be better to add a runtime error if an unrecognized number is provided? Or handle the edge cases explicitly (e.g. error for below zero, but consider anything above 4 "strong")

---

Good catch. Updated to handle unrecognized strength (less than 0) and anything above 4